packages/checkout/sdk/src/smartCheckout/buy/buy.ts

```diff
@@ -87,6 +89,11 @@ export const buy = async (
   config: CheckoutConfiguration,
   provider: Web3Provider,
   orders: Array<BuyOrder>,
+  overrides: {
+    waitFulfillmentSettlements?: boolean;
+  } = {
+    waitFulfillmentSettlements: true,
```

Comment:
wondering if something like `waitOnFulfillmentSettlements` or `waitForFulfillmentSettlments` reads better - i know its already a pretty long name though

Reply:
I don't see the need for adding the additional `On` / `For` -- @dreamoftrees please make the call.

Reply:
Going to keep it short so it doesn't exceed my internal 3 word rule for variables :)